File: CursoPOO/CursoPOOUber/Java/Tarjeta.java
class Tarjeta extends Payment{
    Integer card_number;
    Integer cvv;
    String card_date;
    
    public Tarjeta(String id,Integer card_number, Integer cvv, String card_date){
        super();
        this.card_number = card_number;
        this.cvv = cvv;
        this.card_date = card_date;
    }
    
    void printDataPaymentTarjeta(){
        System.out.println("card_number: " + card_number + " cvv: " + cvv + " card_date: " + card_date);
    }
}

File: CursoPOO/CursoPOOUber/PHP/tarjeta.php
<?php
require_once("payment.php");

class Tarjeta extends Car{
    public $card_num;
    public $card_date;
    public $cvv;

    public function __construct($id, $card_num, $card_date, $cvv){
        parent::__construct($id);
        $this->card_num = $card_num;
        $this->card_date = $card_date;
        $this->cvv = $cvv;
    }
}